networkit/centrality.pyx

```diff
@@ -129,7 +129,8 @@ cdef class Betweenness(Centrality):
 
 
 	def edgeScores(self):
-		""" Get a vector containing the betweenness score for each edge in the graph.
+		""" Get a vector containing the betweenness score for each edge in the graph
+		in ascending edge id order.
```

Comment:
`edge ID`, not `id`.

Reply:
Maybe also add `(as returned by Graph.edgeId())`.

Reply:
@Relux-the-Relux this comment is yet to be addressed.